src/cpp/r/R/Tools.R

```diff
@@ -891,3 +933,13 @@ assign(envir = .rs.Env, ".rs.getVar", function(name)
 .rs.addFunction("nBytes", function(x) {
    nchar(x, type = "bytes")
 })
+
+.rs.addFunction("randomString", function(prefix = "",
```

Comment:
If we do that, we should have a separate function (since this is more flexible) -- I went this route just so I could generate names with a similar format as to what the IDE generates for cache folders already. (I wasn't sure if this is a necessary requirement; likely it isn't but it's nice to be consistent)